### DISCUSSION TITLE
create readme for profile

### DISCUSSION THREAD

**Discussion by akshaypandey89** [community/community#113320] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

how to create profile readme file

**Comment by akashpawar43** [comment]
you can create profile readme file using this website https://readme.so/

**Comment by ettaboyle** [comment]
@akashpawar43 & @akshaypandey89 -- I've seen multiple posts with your accounts asking basic questions and providing answers. This behavior is consistent with Achievements or Badge Hunting. 

**We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges.** You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
